plugins/ProfileExtender/views/addedit.php

```diff
@@ -23,6 +23,7 @@
             </div>
         </div>
     </div>
+    <?php $this->fireEvent('beforeCheckbox'); ?>
```

Comment:
Can we do it with respect of MVC principles. 
Like:
```
<?php $this->data('beforeProfileExtenderCheckbox'); ?>
```
And fire an event before we start to render in:
```
plugins/ProfileExtender/class.profileextender.plugin.php:371

$sender->setData('CurrentFields', $currentFields);
$sender->fireEvent('beforeProfileExtenderAddEditRender'); 
$sender->render('addedit', '', 'plugins/ProfileExtender');
```
This might need changes in implementation when handle in https://github.com/vanilla/internal/pull/1845:
```
plugins/Salesforce/class.salesforce.plugin.php:984

public function settingsController_beforeProfileExtenderAddEditRender_handler($sender) {
$content = '<div...';
$content .= ...
...
$sender->setData('beforeProfileExtenderCheckbox', $content) 
```